pkg/params/validators.go

```diff
@@ -123,3 +123,10 @@ func ValidateSlice(validator ParamValidator) func(value string) error {
 		return nil
 	}
 }
+
+func ValidateNonEmptyString(value string) error {
```

Comment:
I think/hope the `IsMandatory` field of ParamDesc does the same (making sure there's a value).

Reply:
:sweat_smile: you're right, that works!